unlock-app/src/__tests__/pages/pages.test.js

```diff
@@ -59,15 +63,18 @@ describe('Pages', () => {
         privateKey: 'deadbeef',
         balance: '200',
       }
+      const ConfigProvider = ConfigContext.Provider
       rtl.render(
-        <Provider store={store}>
```

Comment:
Previously the global error provider would actually show a "missing account" error here so it was never reaching the point in the react tree where a component needs the config... since the global error provider does not do that anymore, I've had to add the `ConfigProvider` in this test.